### DISCUSSION TITLE
Is it possible to add your own parser via config?

### DISCUSSION THREAD

**Discussion by gertvdb** [tailwindlabs/tailwindcss#11130] [Help]
I want to use this plugin on 'nunjuck' files, which should use probably sort or less use the same parser as normal html files, i guess?
Can i in some way define the extension and map it to the html parser ?

**Comment by wongjn** [comment]
I'm not familiar with Nunjucks, but unless it has some super exotic syntax for declaring class names on html elements, you should not need to configure any special parser for the template files. This is because Tailwind is designed to be language-agnostic – it scans files as string content and does not parse the files with the files' intended code language.

**Reply by gertvdb** [reply]
Thanks for the info. So it should work out of the box when I define .njk files (nunjuck) in my tailwind.config since nunjuck just uses html class. It seems to not do anything in my setup that’s why I asked. But I’ll look into it some more maybe I made a mistake somewhere…
